packages/extension-base/src/background/handlers/Tabs.ts

```diff
@@ -143,9 +144,21 @@ export default class Tabs {
     return null;
   }
 
+  private async redirectIfPhishing (url: string): Promise<boolean> {
+    const { deny } = await this.#state.getPhishingList();
+
+    const isInDenyList = checkHost(deny, url);
```

Comment:
Why not the `checkIfDeny` instead of re-doing the cache logic here?

Reply:
fixed, that's a misunderstanding from my side :)